<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="edtDers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>411</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>öğrenci not hesaplama</string>
   </property>
   <widget class="QTextBrowser" name="txt_sonuc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>öğrenci ad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ders ad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>55</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>vize notu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>final notu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtAd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtDers_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtVize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtFinal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHesapla">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTemizle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>temizle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>çıkış</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnTemizle</sender>
   <signal>clicked()</signal>
   <receiver>txt_sonuc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
